EIPS/eip-9999.md

```diff
@@ -0,0 +1,67 @@
+---
+eip: 9999
```

Comment:
```suggestion
eip: 8012
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.